requirements.txt

```diff
@@ -0,0 +1 @@
+docopt
```

Comment:
is that actually used?

Reply:
@atorkmabrains It will be definitely used in the run scripts. But at that point, I just wanted to add something not to add an empty file for this PR.

Reply:
just adding a `# comment` should be fine but git should also accept empty files.